src/main/scala/org/zalando/spearheads/innkeeper/oauth/OAuthJsonProtocol.scala

```diff
@@ -9,11 +9,11 @@ object OAuthJsonProtocol extends DefaultJsonProtocol {
 
   implicit object ScopeFormat extends RootJsonFormat[Scope] {
 
-    override def write(scope: Scope) = JsString(scope.name)
+    override def write(scope: Scope) = JsArray(scope.scopeNames.map(scopeName => JsString(scopeName)).toVector)
```

Comment:
is `toVector` really necessary? Anyway, `scope.scopeNames.map(JsString(_))` is a bit less verbose